storage/src/object_storage/error.rs

```diff
@@ -27,13 +27,14 @@ use std::io;
 use rusoto_core::RusotoError;
 use rusoto_s3::{
     AbortMultipartUploadError, CompleteMultipartUploadError, CreateMultipartUploadError,
-    DeleteObjectError, GetObjectError, PutObjectError, UploadPartError,
+    DeleteObjectError, GetObjectError, HeadObjectError, PutObjectError, UploadPartError,
 };
 
 use crate::retry::IsRetryable;
 use crate::{StorageError, StorageErrorKind};
 
-pub struct RusotoErrorWrapper<T: StdError>(RusotoError<T>);
+pub struct RusotoErrorWrapper<T: StdError>(pub RusotoError<T>);
```

Comment:
`.source()` returns a `dyn Error` which prevented me for inspecting the inner `RustoError`.